sdk/src/integrationTest/resources

```diff
@@ -0,0 +1 @@
+sdk/src/main/resources/
```

Comment:
I'll need to double check if this is still an issue, but I added this link because integration tests would fail otherwise.

Reply:
`sdk/src/integrationTest/resources` seems like it would be a folder, not a file with text for another resources folder. Perhaps you just need to add the folder with some dummy file?

Reply:
I created a soft link to `sdk/src/main/resources` which is why it appears as a file. I hoped git would be able to show hat nicely, but that doesn't seem to be the case.